packages/graph-explorer/src/connector/sparql/templates/classWithCountsTemplates.ts

```diff
@@ -3,6 +3,7 @@ import dedent from "dedent";
 // It returns the number of instances of the given class
 export default function classWithCountsTemplates(className: string) {
   return dedent`
+    # Fetch the number of instances of the given class
```

Comment:
Why are comments needed in the request body?

Reply:
Mainly because trying to follow along in the browser's network tab is a pain with SPARQL queries. With SPARQL, there can be multiple similar looking queries fired off for a single operation and it is hard to trace back to where they came from in the code.